lib/event.ts

```diff
@@ -1,8 +1,20 @@
-export function getEventName(
-  name: string | string[] | undefined,
-  eventTitle: string,
-  eventNameTemplate: string | null
-) {
-  if (!name || !(typeof name === "string")) name = ""; // If name is not set or is not of proper type
-  return eventNameTemplate ? eventNameTemplate.replace("{USER}", name) : eventTitle + " with " + name;
+import { TFunction } from "next-i18next";
+
+export type EventNameObjectType = {
+  attendee: string | string[] | undefined;
+  eventType: string;
+  eventName: string | null;
+  host: string;
+  t: TFunction;
+};
+
+export function getEventName(eventNameObj: EventNameObjectType) {
+  if (!eventNameObj.attendee || !(typeof eventNameObj.attendee === "string")) eventNameObj.attendee = ""; // If name is not set or is not of proper type
```

Comment:
Not sure about this check. Maybe it would be easier to just check if the attendee name exists, and return something, if not replace the name with `Nameless` or something...

Reply:
Yeah, agreed. I think the check is a bit redundant as well, considering it is being typechecked anyway.

Reply:
>Not sure about this check. Maybe it would be easier to just check if the attendee name exists, and return something, if not replace the name with Nameless or something...
I think this check is more towards satisfying/typefixing the attendee name as "string" in case the name is of type "string[]" which can be sent from `router.query` extraction